packages/labs/ssr-dom-shim/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Let's add a readme for this package.

And somewhere (not sure where) can we mention that in the future this package could move to something like `@webcomponents/ssr-dom-shim`?